plugin/aliasing.go

```diff
@@ -13,22 +13,22 @@ type iamAliaser func(role *gcpRole, svcAccount *iam.ServiceAccount) (alias strin
 type gceAliaser func(role *gcpRole, instance *compute.Instance) (alias string)
 
 const (
-	defaultIAMAlias = "unique_id"
-	defaultGCEAlias = "instance_id"
+	defaultIAMAlias = "role_id"
+	defaultGCEAlias = "role_id"
 )
 
 var (
 	allowedIAMAliases = map[string]iamAliaser{
-		defaultIAMAlias: getIAMSvcAccountUniqueID,
-		"":              getIAMSvcAccountUniqueID, // For backwards compatibility
+		defaultIAMAlias: getIAMRoleID,
+		"":              getIAMRoleID, // For backwards compatibility
```

Comment:
I'm not completely sure that this `""` entry is needed for backwards compatibility anymore now that the default is changing. @pcman312, do you have any context on this?

Reply:
The empty string is there to allow for a value to not be set -> default value. I think that this should remain here and default to the role_id as you have it.

Reply:
That makes sense. I'm just wondering if it's ever used given the logic at https://github.com/hashicorp/vault-plugin-auth-gcp/blob/master/plugin/gcp_config.go#L94. `""` gets replaced with the default key.

Reply:
Probably not - If I recall correctly I put the empty string check in as an over-abundance of caution. If we want to remove one, I'd prefer to remove the empty string check so the map is a single place where we see all of the values + default together.

Reply:
Got it 👍 . Done in 74e03c93fa7577ea9caa1d6fb9127ed7ae83a811.

Reply:
Reran tests successfully as well.